meson.build

```diff
@@ -11,14 +11,14 @@ project(
         'warning_level=2',
         'force_fallback_for=xxhash,lz4,cjson',
     ],
-    meson_version: '>=0.58.0',
```

Comment:
hse-java is moving to 0.62.

I am just using a venv for now, which is what we have done in the past.

`python3 -m venv /opt/venv; source /opt/venv/bin/activate; python3 -m pip install meson`

Reply:
can we have that set up by cobbler?  it's probably a non-starter for most folk to require software that even the most bleeding edge distro doesn't support (IMO it's a PITA).

Reply:
@bhaveshvasandani would know whether it is doable or not. Arch already has it :P. https://archlinux.org/packages/extra/any/meson/.

Fedora is a PITA because it doesn't update Meson at all.